### ISSUE TITLE
Sidebar is not flush with the bottom of the screen

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#1236]
**Steps to Reproduce**
- Visit a page that has long content (e.g., the brick reference, the marketplace search results, etc)

**Behavior**
- The full page has a scroll, and the sidebar ends so there's a background color gap

The scroll bar is on the full page, and is under the top nav

![image](https://user-images.githubusercontent.com/1879821/131587111-0a13c91a-3ed6-4c99-a20f-4818cfd50b0f.png)

![image](https://user-images.githubusercontent.com/1879821/131586869-5d6891f7-a308-4c38-9f77-da7f622cc839.png)

**Comment by fregante** [comment]
### Reason

https://github.com/pixiebrix/pixiebrix-extension/blob/4b95d7d396bdcd585b9676527f8a4021efda0dce/src/vendors/theme/assets/styles/_sidebar.scss#L2

### Quick fix

```css
.page-body-wrapper.container-fluid > div {
	display: flex
}
```

This fixes it, but the selector is probably improvable.


### Long-term solution

- https://github.com/pixiebrix/pixiebrix-extension/issues/1234

There should be **one** wrapper that extends to "min-height: 100vh" and then everything inside it will stretch to fill it via flex box.

**Comment by BALEHOK** [comment]
Another option is to fix the sidebar. Items are always visible and it fits the screen.
See PR https://github.com/pixiebrix/pixiebrix-extension/pull/1257

**Comment by fregante** [comment]
> There should be **one** wrapper that extends to "min-height: 100vh" and then everything inside it will stretch to fill it via flex box.

Looks like you took this approach 👍

**Comment by BALEHOK** [comment]
> Looks like you took this approach 👍
Well, kind of... The layout is a little bit messed up now, so #1234 (nice ID btw) definitely needs attention. We should review and tidy up global page layout, positioning, and scroll areas.

**Comment by BLoe** [comment]
Closed via #1257
